package.json

```diff
@@ -27,17 +27,17 @@
   },
   "dependencies": {
     "@types/eslint": "^8.4.1",
-    "@typescript-eslint/eslint-plugin": "^6.5.0",
-    "@typescript-eslint/parser": "^6.5.0",
+    "@typescript-eslint/eslint-plugin": "^6.13.2",
+    "@typescript-eslint/parser": "^6.13.2",
     "eslint-config-seek": "^12.0.1",
-    "eslint-plugin-jest": "^27.0.0",
+    "eslint-plugin-jest": "^27.6.0",
```

Comment:
Saw that we missed this potential peer dep fix: https://github.com/jest-community/eslint-plugin-jest/releases/tag/v27.2.3